transfer-products/cloudformation.yml

```diff
@@ -50,17 +57,17 @@ Resources:
           HYP3_URL: !Ref HyP3URL
           EARTHDATA_USERNAME: !Ref EarthdataUsername
           EARTHDATA_PASSWORD: !Ref EarthdataPassword
+          S3_TARGET_BUCKET: !Ref S3TargetBucket
           S3_TARGET_PREFIX: !Ref S3TargetPrefix
       Code: transfer-products/src/
       Handler: transfer_products.lambda_handler
       MemorySize: 128
       Role: !GetAtt Role.Arn
       Runtime: python3.9
       Timeout: 900
-      # TODO if we migrate to an enterprise test deployment: remove this property
       EphemeralStorage:
         # TODO decide on appropriate size
-        Size: 10240 # MB
+        Size: 1024 # MB
```

Comment:
We're still downloading the entire file to disk before uploading to S3, so I'm suspicious that 1 GB disk will be sufficient.